### DISCUSSION TITLE
START_PRINT in macros.cfg

### DISCUSSION THREAD

**Discussion by CaryBab** [Guilouz/Klipper-Flsun-Speeder-Pad#114] [General]
I don't know if it is intentional, but the statement to load the bed mesh is not there.

**Comment by Guilouz** [comment]
Yes it's normal. My configuration use a macro to load the default bed mesh at Klipper starting.

**Comment by CaryBab** [comment]
Ahh, I see it now, a delayed gcode.  Thanks.
